spec/rails_app/app/javascript/application.js

```diff
@@ -0,0 +1,10 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
```

Comment:
We can probably remove the import maps-rails comment

Reply:
Done :)